### PR TITLE
feat: implement `r/demo/postit`

### PR DESCRIPTION
**Pull request by harry-hov**
Very Basic Twitter Clone 

Frontend: https://postitsocial.netlify.app/

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: examples/gno.land/r/demo/postit/json.gno [hunk 1]
```diff
@@ -0,0 +1,188 @@
+package postit
```
Comment (moul) [inline examples/gno.land/r/demo/postit/json.gno:1]: Include a comment stating this file is temporary, pending JSON support implementation.

File: examples/gno.land/r/demo/postit/user.gno [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	}
+	res, ok := users.Get(username.(string))
+	if !ok {
+		panic("user with username not found")
+	}
+	return res.(*User)
+}
+
+func getUserRefByUsername(username string) *User {
+	res, ok := users.Get(username)
```
Comment (moul) [inline examples/gno.land/r/demo/postit/user.gno:34]: Would be nice to be compatible with r/users; or maybe having r/users to be the way to get “verified” while having a built in light alternative?

File: examples/gno.land/r/demo/postit/json.gno (hunk 1 above)
Comment (moul) [inline examples/gno.land/r/demo/postit/json.gno:1]: What about adding a Render() func too, could just give some basic info and suggest to use the better frontend.

**Comment by thehowl** [comment]
Unlikely to be continued; if someone wants to continue it please open a separate PR continuing Hariom's work.

**Pull request closed without merging**